compiler/testdata/coroutine_durable.go

```diff
@@ -3384,5 +3438,11 @@ func init() {
 	_types.RegisterFunc[func()]("github.com/stealthrocket/coroutine/compiler/testdata.YieldingExpressionDesugaring")
 	_types.RegisterFunc[func(_fn0 int) (_ int)]("github.com/stealthrocket/coroutine/compiler/testdata.a")
 	_types.RegisterFunc[func(_fn0 int) (_ int)]("github.com/stealthrocket/coroutine/compiler/testdata.b")
+	_types.RegisterFunc[func(n int) func()]("github.com/stealthrocket/coroutine/compiler/testdata.buildClosure[go.shape.int]")
+	_types.RegisterClosure[func(), struct {
+		F  uintptr
+		D  uintptr
```

Comment:
Interesting, so the generic closures hold a pointer to the dictionary as the second field, I misunderstood that detail last time.